sql/analyzer/pushdown_test.go

```diff
@@ -228,5 +228,5 @@ func TestPushdownIndexable(t *testing.T) {
 	})
 	require.NoError(err)
 
-	require.Equal(expected, result)
+	require.Equal(expected, removeProcessNodes(t, result))
```

Comment:
I would prefer to add process node to expected or remove the rule to the actual analyzer. WDYT?

Reply:
It can't be added to the expected, because the nodes have functions and they're not comparable, so I guess I can remove it from the analyzer